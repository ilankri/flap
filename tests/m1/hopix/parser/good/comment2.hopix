
-- This is also a comment.

val x = 1 + -- Another comment
1

val x = 1 +-- Still a comment
2

val x = 3-- Again...

-- A final comment --
