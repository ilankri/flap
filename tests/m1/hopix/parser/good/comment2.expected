
val x = `+ (1, 1)
val x = `+ (1, 2)
val x = 3
